client/src/routes/HereMaps/index.jsx

```diff
@@ -174,94 +321,182 @@ const HereMaps = () => {
   return (
     <>
       <div className="page-header">
-        <h1>Congestion and PM2.5</h1>
-        <button
-          onClick={() => {
-            return setShowPm((prevState) => {
-              return !prevState;
-            });
-          }}
-        >
-          {showPm ? "Show Congestion" : "Show PM2.5"}
-        </button>
-        <br />
-        {routes ? (
-          <>
-            <button
+        {/* <div>{inval}</div> */}
+        <div id="appbar">
+          <div ref={wrapperRef}>
+            <div class="input">
```

Comment:
change all occurrences of `class` to `className`, also remove other such errors in browser console